pkg/k8shandler/fluentd.go

```diff
@@ -25,8 +25,10 @@ import (
 )
 
 const (
-	fluentdAlertsFile = "fluentd/fluentd_prometheus_alerts.yaml"
-	fluentdName       = "fluentd"
+	fluentdAlertsFile          = "fluentd/fluentd_prometheus_alerts.yaml"
+	fluentdName                = "fluentd"
+	fluentdTrustedCABundleName = "fluentd-trusted-ca-bundle"
```

Comment:
lets consolidate this into a single var -- no reason to have two different constants with the same value. `fluentdTrustedCAName` ?